### ISSUE TITLE
Issue with Markdown style links.

### ISSUE THREAD

**Issue by PhilipMay** [tsproisl/SoMaJo#27]
Links in this format: `"MD link <https://heise.de> example."` have an issue.

Code:
```python
text = "MD link <https://heise.de> example."
sentences = somajo.tokenize_text([text])
for sentence in sentences:
    for token in sentence:
        print(f"{token.text}\t{token.token_class}\t{token.extra_info}")
```

Returns:
```
MD	regular	
link	regular	
<	symbol	SpaceAfter=No
https://heise.de/>	URL	
example	regular	SpaceAfter=No
.	symbol
```

Should return something like this:
```
MD	regular	
link	regular	
<	symbol	SpaceAfter=No
https://heise.de/	URL
>	symbol
example	regular	SpaceAfter=No
.	symbol
```

Full code: https://colab.research.google.com/drive/16-CKdzp20Gin02emrLVeHfFFir2veK8M?usp=sharing

**Comment by tsproisl** [comment]
This should be closely related to #26 and can probably be fixed in the same manner. I’ll see if I can take a closer look tomorrow.

**Comment by tsproisl** [comment]
It should be fixed in version 2.4.1!

**Comment by PhilipMay** [comment]
it is fixed. Many thanks.
